common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -45,9 +45,9 @@ class VerticalBlock(SequenceFields, XModuleFields, StudioEditableBlock, XmlParse
 
     show_in_read_only_mode = True
 
-    def student_view(self, context):
+    def _student_view(self, context, view):
```

Comment:
Nit: It's a little confusing to have a `_student_view` method that can accept a `view` parameter that's `STUDENT_VIEW`. Maybe it'd be clearer to rename the method or to replace `view` with something like a boolean `access_user_state=True`?

Reply:
Ah yes, will update.

Reply:
Since `view` parameter is passed down to the children, it makes sense to have it passed in from the views. I have updated the name of this method to `_student_or_public_view()` instead.